server/controllers/request.js

```diff
@@ -0,0 +1,70 @@
+const Request = require("../models/Request");
+const asyncHandler = require("express-async-handler");
+
+// @route GET /requests
+// @desc get requests for logged in user
+// @access Private
+exports.getRequests = asyncHandler(async (req, res) => {
+  const requests = await Request.find({
+    sitterId: req.user.id,
+  });
+
+  res.status(200).json({
+    success: { requests },
+  });
+});
+
+// @route POST /requests
+// @desc Create a new request
+// @access Private
+exports.createRequest = asyncHandler(async (req, res) => {
+  const { sitterId, start, end } = req.body;
+  const request = await Request.create({
+    userId: req.user.id,
+    sitterId,
+    start: start,
+    end: end,
+    status: "pending",
+  });
+
+  if (!request) {
+    res.status(400);
```

Comment:
```suggestion
    res.status(500);
```
A `500` status code seems more appropriate here!

Reply:
My thinking was that those lines would validate whether or not all of the required fields were provided in the body but I was wrong. So I agree that it should be a 500